zebra/zebra_ptm.c

```diff
@@ -589,6 +589,8 @@ static int zebra_ptm_handle_msg_cb(void *arg, void *in_ctxt)
 	char port_str[128];
 	char cbl_str[32];
 	char cmd_status_str[32];
+	char vrf_str[64];
```

Comment:
`VRF_NAMSIZ` ?

Reply:
not sure, as initially in BFD, the vrf name size is not using the same length as vrf name.
also, as the code is used to communicate with the cumulus ptm, I am curious to know whether this breaks something or not.

Reply:
I'd like this figured out

Reply:
2 changes:
- I use VRF_NAMSIZ + 1 ( like in vrf.h)
- bfd MAXNAMELEN value changed with VRF_NAMSIZ + 1 ( instead of 32)